core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go

```diff
@@ -533,6 +533,14 @@ lookbackBlocks         	= %d # This is an integer
 	// poll until we're able to redeem the randomness without reverting
 	// at that point, it's been fulfilled
 	gomega.NewWithT(t).Eventually(func() bool {
+		// Ensure a refund is provided. Refund amount comes out to ~23 million Gwei.
+		var refundAmount int64 = 23_822_200
+		subAfterRefund, err := uni.coordinator.GetSubscription(nil, 1)
+		require.NoError(t, err)
+		if subAfterRefund.Balance.Int64() != subAfterBatchFulfillmentRequest.Balance.Int64()+assets.GWei(refundAmount).Int64() {
+			return false
+		}
```

Comment:
Should this be an assertion/require? Or is this basically the indication that all requests have been billed and refunded the appropriate amount?

Reply:
Also, probably better to just stick to using `big.Int` operations since we're using such big numbers here:

```
expected := new(big.Int).Add(subAfterBatchFulfillmentRequest.Balance, assets.GWei(refundAmount))
actual := subAfterRefund.Balance
if expected.Cmp(actual) != 0 {
  return false
}
```